Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -14,14 +14,35 @@
 @implementation BNCServerInterface
 
 // make a generalized get request
-- (void)getRequestAsync:(NSDictionary *)params url:(NSString *)url andTag:(NSString *)requestTag {
-    [self getRequestAsync:params url:url andTag:requestTag log:YES];
+- (void)getRequestAsync:(NSDictionary *)params url:(NSString *)url andTag:(NSString *)requestTag retryCount:(NSInteger)retryCount {
+    [self getRequestAsync:params url:url andTag:requestTag retryNumber:0 retryCount:retryCount log:YES];
 }
 
 // this is actually a synchronous call; it should NOT be called from the main queue
```

Comment:
Fortunately these are internal methods, not really public facing. I agree that they are confusingly named, though, and have plans to revisit them with the callback structure we had discussed.